{{ cookiecutter.municipality_slug }}/python/example_scraper.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import random
+from datetime import datetime
+from typing import List
+
+from cdp_backend.pipeline.ingestion_models import (
+    Body,
+    EventIngestionModel,
+    EventMinutesItem,
+    Matter,
+    MinutesItem,
+    Person,
+    Role,
+    Seat,
+    Session,
+    SupportingFile,
+    Vote,
+)
+
+###############################################################################
+
+
+def get_example_events(
+    n: int = 10, is_minimal: bool = False
+) -> List[EventIngestionModel]:
+    return [_get_example_event(is_minimal) for _ in range(n)]
+
+
+###############################################################################
+
+
+FAKE_COUNCIL_SEAT_NUM = 10
```

Comment:
1. nitpick: can we change this to `NUM_COUNCIL_SEATS`
2.
I would make all the range values constants here as well.

```python
RAND_BODY_RANGE = (1, 100)
RAN_EVENTS_MINUTES_ITEMS_RANGE = (5, 15)
```

etc.